### PR TITLE
[libc] Updated GettingStarted.rst with PyYAML version

### PR DESCRIPTION
**Pull request by RoseZhang03**
New Headergen requires PyYAML version 5.1 or newer in order to generate
header files .

### CODE REVIEW

File: llvm/docs/GettingStarted.rst [hunk 1]
```diff
@@ -295,6 +295,7 @@ Package                                                     Version      Notes
 `python <http://www.python.org/>`_                          >=3.8        Automated test suite\ :sup:`1`
```
Comment (boomanaiden154) [inline llvm/docs/GettingStarted.rst:295]: Side note: This looks like it's different from what is posted at https://github.com/llvm/llvm-project/blob/888ae9066da9c9758c03e839be6cefb3246a9b1c/libc/docs/dev/header_generation.rst?plain=1#L25? Should we just adjust the version on the header generation page to say 3.8 given that is the LLVM project-wide minimum?

Comment (lntue) [reply llvm/docs/GettingStarted.rst:295]: SGTM

Comment (RoseZhang03) [reply llvm/docs/GettingStarted.rst:295]: Yes, I can edit the header generation page to match

**Review by aaryanshukla — APPROVED** [review]
LGTM!

**Review by lntue — APPROVED** [review]

**Review by boomanaiden154 — APPROVED** [review]

**Pull request merged**